Client/src/Pages/Monitors/index.jsx

```diff
@@ -152,17 +154,52 @@ const Monitors = () => {
 
   const down = monitorState.monitors.length - up;
 
+  const [sortOrder, setSortOrder] = useState(-1);
+  const [sortedMonitors, setSortedMonitors] = useState(monitorState.monitors);
+  const [isSorted, setIsSorted] = useState(false);
+
+  useEffect(() => {
+    const savedSortOrder = localStorage.getItem('monitorSortOrder');
+    const parsedSortOrder = parseInt(savedSortOrder, 10) || 1;
+  
+    if (savedSortOrder) {
+      setSortOrder(parsedSortOrder);
+      const monitors = [...monitorState.monitors].sort(a => {
+        return a.status ? -1 * parsedSortOrder : 1 * parsedSortOrder;
+      });
+      setSortedMonitors(monitors);
+      setIsSorted(true);
+    } else {
+      setSortedMonitors(monitorState.monitors);
+    }
+  }, [monitorState.monitors]);
```

Comment:
I think we can just drop this entirely, I don't think we need to persist the sort order through reloads of the page.

If we _did_ decide we wanted to do that we should use the Redux state for that in any case.

Fow now, I think safe to just drop this